ci/ray_ci/docker_container.py

```diff
@@ -0,0 +1,44 @@
+import os
+
+from ci.ray_ci.container import Container, _DOCKER_ECR_REPO
+from ci.ray_ci.builder_container import PYTHON_VERSIONS
+from ci.ray_ci.utils import docker_pull
+
+
+class DockerContainer(Container):
+    def __init__(self, python_version: str) -> None:
+        super().__init__(
+            "forge",
+            volumes=[
+                f"{os.environ['RAYCI_CHECKOUT_DIR']}:/rayci",
+                "/var/run/docker.sock:/var/run/docker.sock",
+            ],
+        )
+        self.python_version = python_version
+
+    def run(self) -> None:
+        base_image = (
+            f"{_DOCKER_ECR_REPO}:{os.environ['RAYCI_BUILD_ID']}-raypy38cu118base"
+        )
+        docker_pull(base_image)
+        wheel_name = (
+            "ray-3.0.0.dev0-"
+            f"{PYTHON_VERSIONS[self.python_version]['bin_path']}-"
+            "manylinux2014_x86_64.whl"
+        )
+        constraints_file = (
+            "requirements_compiled_py37.txt"
+            if self.python_version == "py37"
+            else "requirements_compiled.txt"
+        )
```

Comment:
I always find this a bit hard to understand.. could you change it to a plain:

```
if xxx:
  x = one
else:
  x = two
```

it even take fewer lines..

or 

```
x = default_value
if xxx: x = alternative_value;
```